config.c

```diff
@@ -138,7 +138,7 @@ int tabooCount = 0;
 static int glob_errno = 0;
 
 static int readConfigFile(const char *configFile, struct logInfo *defConfig);
-static int globerr(const char *pathname, int theerr);
+static int globerr(const char *pathname __attribute__((__unused__)), int theerr);
```

Comment:
Attribute usage is done with conditional definition similar to what one can find from gnulib. I am sure that's good enough compromise.